config/projects/amp-ad-project.yaml

```diff
@@ -22,3 +22,4 @@ stack_tags:
   Department: NDR
   Project: amp-ad
   OwnerEmail: william.poehlman@sagebase.org
+  SageFinanceProgramCode: NIA AMP-AD CC | 10015
```

Comment:
would be nice to make stack_tags resolvable (i.e. https://github.com/Sceptre/sceptre/pull/847), that way we could pull all these tags in from some type of spreadsheet.

also, not a big deal, the term AWS likes to use for this is `CostCenter` https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html

Reply:
@brucehoff: What do you think about `SageFinanceProgramCode` vs `CostCenter`?